tests/CI/pilot_ci.sh

```diff
@@ -101,6 +101,10 @@ PilotInstall(){
   if [[ "${modules}" ]]; then
     pilotOptions+=" --modules=""${modules}"
   fi
+  # shellcheck disable=SC2154
+  if [[ "${pip_install_options}" ]]; then
+    pilotOptions+=" --pipInstallOptions=""${pip_install_options}"
```

Comment:
This looks fishy with the redundant `""` in the middle, do you mean to do this:
```suggestion
    pilotOptions+=" --pipInstallOptions=${pip_install_options}"
```

Reply:
Or do you mean this?
```suggestion
    pilotOptions+=" --pipInstallOptions='${pip_install_options}'"
```

Reply:
Fixed thanks.